src/openforms/contrib/objects_api/setup_configuration/tests/test_objects_api_config.py

```diff
@@ -0,0 +1,224 @@
+        )
+        self.drc_service = ServiceFactory.create(
+            slug="documenten-test",
+            label="Documenten API test",
+            api_root="http://localhost:8003/documenten/api/v1/",
+            api_type=APITypes.drc,
+            auth_type=AuthTypes.zgw,
+            client_id="test_client_id",
+            secret="test_secret_key",
+        )
+        self.catalogi_service = ServiceFactory.create(
+            slug="catalogi-test",
+            label="Catalogi API test",
+            api_root="http://localhost:8003/catalogi/api/v1/",
+            api_type=APITypes.ztc,
+            auth_type=AuthTypes.zgw,
+            client_id="test_client_id",
+            secret="test_secret_key",
+        )
+
+    def test_execute_success(self):
+        execute_single_step(ObjectsAPIConfigurationStep, yaml_source=CONFIG_FILE_PATH)
+
+        self.assertEqual(ObjectsAPIGroupConfig.objects.count(), 2)
+
+        config1, config2 = ObjectsAPIGroupConfig.objects.all()
+
+        self.assertEqual(config1.name, "Config 1")
+        self.assertEqual(config1.identifier, "config-1")
+        self.assertEqual(config1.objects_service, self.objects_service)
+        self.assertEqual(config1.objecttypes_service, self.objecttypes_service)
+        self.assertEqual(config1.drc_service, self.drc_service)
+        self.assertEqual(config1.catalogi_service, self.catalogi_service)
+        self.assertEqual(config1.catalogue_domain, "TEST")
+        self.assertEqual(config1.catalogue_rsin, "000000000")
+        self.assertEqual(config1.organisatie_rsin, "000000000")
+        self.assertEqual(config1.iot_submission_report, "PDF Informatieobjecttype")
+        self.assertEqual(config1.iot_submission_csv, "CSV Informatieobjecttype")
+        self.assertEqual(config1.iot_attachment, "Attachment Informatieobjecttype")
+
+        self.assertEqual(config2.name, "Config 2")
+        self.assertEqual(config2.identifier, "config-2")
+        self.assertEqual(config2.objects_service, self.objects_service)
+        self.assertEqual(config2.objecttypes_service, self.objecttypes_service)
+        self.assertEqual(config2.drc_service, self.drc_service)
+        self.assertEqual(config2.catalogi_service, self.catalogi_service)
+        self.assertEqual(config2.catalogue_domain, "OTHER")
+        self.assertEqual(config2.catalogue_rsin, "000000000")
+        self.assertEqual(config2.organisatie_rsin, "000000000")
+        self.assertEqual(config2.iot_submission_report, "")
+        self.assertEqual(config2.iot_submission_csv, "")
+        self.assertEqual(config2.iot_attachment, "")
+
+    def test_execute_update_existing_config(self):
+        ObjectsAPIGroupConfigFactory.create(name="old name", identifier="config-1")
+
+        execute_single_step(ObjectsAPIConfigurationStep, yaml_source=CONFIG_FILE_PATH)
+
+        self.assertEqual(ObjectsAPIGroupConfig.objects.count(), 2)
+
+        config1, config2 = ObjectsAPIGroupConfig.objects.all()
```

Comment:
the ordering here is arbitrary and up to the DB configuration and how the underlying storage tuples get stored. I'd make this explicit to prevent flakiness:

```suggestion
        config1, config2 = ObjectsAPIGroupConfig.objects.order_by("pk")
```